src/gwt/src/org/rstudio/studio/client/workbench/views/jobs/view/JobProgress.java

```diff
@@ -63,26 +77,39 @@ public void showJob(Job job)
       name_.setText(job.name);
       progress_.setLabel(job.name);
       String status = JobConstants.stateDescription(job.state);
+      stop_.addClickHandler(new ClickHandler()
```

Comment:
Minor style thing: new code can use lambda syntax instead of anonymous class, so:

```
stop_.addClickHandler(clickEvent -> eventBus_.fireEvent(new JobExecuteActionEvent(job.id, JobConstants.ACTION_STOP));
```